released/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.1.5/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+name: "coq-mathcomp-multinomials"
+maintainer: "pierre-yves@strub.nu"
+homepage: "https://github.com/math-comp/multinomials"
+bug-reports: "https://github.com/math-comp/multinomials/issues"
+dev-repo: "git+https://github.com/math-comp/multinomials.git"
+license: "CeCILL-B"
+authors: ["Pierre-Yves Strub"]
+build: [
+  [make "INSTMODE=global" "config"]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "coq" {(>= "8.7" | < "8.11~")}
```

Comment:
```suggestion
  "coq" {(>= "8.7" & < "8.11~")}
```
Regretfully I didn't spot this right away, but this is supposed to be a closed range, right? Unless I'm mistaken, the current constraints allow even 8.5 and earlier.